src/SourceBrowser/src/SourceIndexServer/SourceIndexServer.csproj

```diff
@@ -10,14 +10,12 @@
   <ItemGroup>
     <PackageReference Include="Azure.Identity" />
     <PackageReference Include="Azure.Storage.Blobs" />
+    <PackageReference Include="Microsoft.Extensions.Hosting.WindowsServices" />

```

Comment:
I know you didn't add this one on this PR, but do we actually need it? What breaks if this is removed?

Reply:
No idea, I just moved the existing packageref around when cleaning up my changes to the csproj